packages/grid/src/vaadin-grid-drag-and-drop-mixin.js

```diff
@@ -192,6 +194,12 @@ export const DragAndDropMixin = (superClass) =>
         });
         event.originalEvent = e;
         this.dispatchEvent(event);
+
+        requestAnimationFrame(() => {
+          rows.forEach((row) => {
+            updateCellsPart(getBodyRowCells(row), 'dragstart-source-row-cell', true);
+          });
```

Comment:
We could probably use `updateBooleanRowStates` for this as well (also for the state removal). It also applies a state attribute but that shouldn't be an issue.